javascript/javascript1/week2/index.html

```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body>
+    <script src="java.js"></script>
```

Comment:
Java and JavaScript are two different programming languages :) 

https://en.wikipedia.org/wiki/JavaScript

https://en.wikipedia.org/wiki/Java_(programming_language)